src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -207,6 +253,41 @@ function createModel(program: Program): any {
     }
 }
 
+function createContentTypeOrAcceptParameter(
+    mediaTypes: string[],
+    name: string,
+    nameInRequest: string
+): InputParameter {
+    const isContentType: boolean =
+        nameInRequest.toLowerCase() === "content-type";
+    const inputType: InputType = {
+        Name: "String",
+        Kind: InputTypeKind.String,
+        IsNullable: false
+    } as InputPrimitiveType;
+    return {
+        Name: name,
+        NameInRequest: nameInRequest,
+        Type: inputType,
+        Location: RequestLocation.Header,
+        IsApiVersion: false,
+        IsResourceParameter: false,
+        IsContentType: isContentType,
+        IsRequired: true,
+        IsEndpoint: false,
+        SkipUrlEncoding: false,
+        Explode: false,
+        Kind: InputOperationParameterKind.Constant,
+        DefaultValue:
+            mediaTypes.length === 1
```

Comment:
Shall we still use     mediaTypes.length >= 1  
So if there are multiple values, we pick the first one.

Reply:
No. when it has multiple types, no default value will be set.